addons/stock/wizard/make_procurement.py

```diff
@@ -64,9 +65,14 @@ def make_procurement(self):
         """ Creates procurement order for selected product. """
         ProcurementOrder = self.env['procurement.order']
         for wizard in self:
+            # we set the time to noon to avoid the date to be changed because of timezone issues
+            date = datetime.datetime.strptime(wizard.date_planned, DEFAULT_SERVER_DATE_FORMAT)
+            date = date + datetime.timedelta(hours=12)
+            date = date.strftime(DEFAULT_SERVER_DATETIME_FORMAT)
```

Comment:
Why not using `fields.Datetime.to_string`?

Reply:
Because I didn't know these functions. Thanks for the tip!